tests/normal.t.pl

```diff
@@ -28,10 +28,10 @@ sub runTests
     my $hardlinks = 1;
     if($mode eq 'standard')
     {
-        &mount("--standard");
+        &mount("--standard -o direct_io");
     } elsif($mode eq 'paranoia')
     {
-        &mount("--paranoia");
+        &mount("--paranoia -o direct_io");
```

Comment:
(This works on Linux because, by default, Linux flushes the file from cache when it is opened. Check out kernel_cache in `man mount.fuse`.)